app/mailers/mailbot.rb

```diff
@@ -80,6 +80,20 @@ def send_on_call_for_papers_dates_updates(conference)
     end
   end
 
+  def notification_email(conference, event, comment)
+    recipients = User.joins(:roles).where('roles.name IN (?)', [:organizer, :cfp]).where('roles.resource_id = ?', conference.id).select(:name, :email).distinct
```

Comment:
I'd prefer to use a scope here. Not sure if there already exists one for organizer / cfp but we could for sure add one.

http://guides.rubyonrails.org/active_record_querying.html#scopes